configs/projects/puppet-agent.rb

```diff
@@ -1,34 +1,63 @@
 project "puppet-agent" do |proj|
-  # Project level settings our components will care about
-  proj.setting(:install_root, "/opt/puppetlabs")
-  proj.setting(:prefix, File.join(proj.install_root, "puppet"))
+  platform = proj.get_platform
 
-  if platform.is_eos?
-    proj.setting(:sysconfdir, "/persist/sys/etc/puppetlabs")
-    proj.setting(:link_sysconfdir, "/etc/puppetlabs")
-  elsif platform.is_osx?
-    proj.setting(:sysconfdir, "/private/etc/puppetlabs")
+  # Project level settings our components will care about
+  # Windows has its own seperate layout
+  if platform.is_windows?
+    proj.setting(:install_root, "#{platform.drive_root}/ProgramFiles/PuppetLabs")
```

Comment:
@glennsarti good point. I was assuming we're always building 32-bit MSIs on 32-bit Windows and building 64-bit MSIs on 64-bit Windows, but I think that's a bad assumption, since I think we always build on 2012r2. So yeah ProgFiles (x86) vs ProgFiles could be an issue. It appears that ~1 maps to ProgFiles and ~2 maps to ProgFiles (x86):

```
Administrator@mm1fxccjkyk8b1h /cygdrive/c
$ cygpath -ad "Program Files"
C:\PROGRA~1

Administrator@mm1fxccjkyk8b1h /cygdrive/c
$ cygpath -ad "Program Files (x86)"
C:\PROGRA~2
```

I would expect that to always be the case, but maybe not?